src/openloco/graphics/colour.cpp

```diff
@@ -0,0 +1,24 @@
+#include "../interop/interop.hpp"
+#include "colours.h"
+#include <cassert>
+
+using namespace openloco::interop;
+
+namespace openloco::colour
+{
+
+    loco_global<uint8_t[32][8], 0x01136BA0> _colour_map_a;
+    loco_global<uint8_t[32][8], 0x01136C98> _colour_map_b;
+
+    uint8_t get_shade(colour_t colour, uint8_t shade)
```

Comment:
Can we not just use the same structures as rct. That's what i was planning on doing as viewport initialise also plays with the colour maps.

Reply:
Not a fan of that idea. `ColourMapA` is used so frequently without bounds checking, and the member names don't really make any sense. Plus that loco still has the two arrays seperately.

Reply:
In rct2 we had the two structures originally. Then they were combined into one later on.